index.html

```diff
@@ -220,6 +220,25 @@ <h3 class="small__resume_title">Skills</h3>
         </section>
 
         <!-- Contact Form -->
+        <section class="contacts">
+            <h2 class="contact_heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="" class="contact_form" method="post">
```

Comment:
Kindly add an action using the [formspree service](https://formspree.io/html) 🤗

- `action` value should be similar to `https://formspree.io/f/{form_id}`